.github/workflows/release.yml

```diff
@@ -32,3 +32,48 @@ jobs:
           workflow: publish.yml
           ref: master
           inputs: '{ "acvm-ref": "${{ needs.release-please.outputs.tag-name }}" }'
+
+  update-lockfile:
+    name: Update lockfile
+    needs: [release-please]
+    if: ${{ needs.release-please.outputs.release-pr }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout release branch
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ fromJSON(needs.release-please.outputs.release-pr).headBranchName }}
+          token: ${{ secrets.ACVM_SIMULATOR_RELEASES_TOKEN }}
+
+      - name: Setup toolchain
+        uses: dtolnay/rust-toolchain@1.66.0
+
+      - name: Update lockfile
+        run: |
+          cargo update --workspace
+
+      - name: Configure git
+        run: |
+          git config user.name tomafrench
+          git config user.email tomfrench@aztecprotocol.com
+
+      - name: Commit updates
+        run: |
+          git add Cargo.lock
+          git commit -m 'chore: Update lockfile'
+          git push
```

Comment:
This can be removed if we're getting rid of `acvm_js/Cargo.lock`

Reply:
addressed in [#127657d](https://github.com/noir-lang/acvm/pull/459#:~:text=37%20minutes%20ago-,chore%3A%20drops%20update%2Dlockfile%20from%20release,-127657d)